core/src/main/java/com/zextras/carbonio/files/dal/repositories/interfaces/NodeRepository.java

```diff
@@ -306,4 +306,28 @@ Optional<Node> getNodeByName(
     String folderId,
     String nodeOwner
   );
+
+  /**
+   * @param ownerId is the owner id from which to return nodes.
+   *
+   * <p>Returns a {@link List<Node>} representing all nodes owned by ownerId.</p>
```

Comment:
the javadoc format has a specific keyword to describe the return part of a method
```suggestion
   * @return a {@link List<Node>} representing all nodes owned by ownerId.</p>
```